### ISSUE TITLE
`npm run update-modules` not working properly

### ISSUE THREAD

**Issue by jmMeessen** [zeon-studio/hugoplate#72]
As mentioned in https://github.com/zeon-studio/hugoplate/issues/68#issuecomment-1874612105 , the command `npm run update-modules` fails with `go: malformed module path "hugoplate": missing dot in first path element`. 

I cloned a fresh copy of the hugoplate repo, ran `npm run project-setup` and then `npm install`.  But the update-modules still fails with the same error 😞 

I am using up to date prerequisite I believe:

- hugo v0.121.1-00b46fed8e47f7bb0a85d7cfc2d9f1356379b740+extended darwin/amd64 BuildDate=2023-12-08T08:47:45Z VendorInfo=brew
- go version go1.21.5 darwin/amd64

```
npm version
{
  hugoplate: '1.10.3',
  npm: '10.2.4',
  node: '21.4.0',
  acorn: '8.11.2',
  ada: '2.7.4',
  ares: '1.23.0',
  base64: '0.5.1',
  brotli: '1.1.0',
  cjs_module_lexer: '1.2.2',
  cldr: '43.1',
  icu: '73.2',
  llhttp: '9.1.3',
  modules: '120',
  napi: '9',
  nghttp2: '1.58.0',
  openssl: '3.2.0',
  simdutf: '4.0.4',
  tz: '2023c',
  undici: '5.28.1',
  unicode: '15.0',
  uv: '1.47.0',
  uvwasi: '0.0.19',
  v8: '11.8.172.17-node.17',
  zlib: '1.2.13.1-motley-5daffc7'
}
```

Any idea where to search to fix the theme update? Where is the theme module defined ?

**Comment by tfsomrat** [comment]
hey @jmMeessen, sorry to hear that. can you please tell me which OS you are using?

**Comment by tfsomrat** [comment]
I am on the same OS, and I have downloaded your repo and tried it on my machine and some other machines. and everything works expectedly. I don't understand why it's not working on your machine.

**Comment by jmMeessen** [comment]
So we can exclude something with the theme's code but rather on my two laptops. 

Just in case, are you using bash or zsh as default shell?

( I will the new update script during my lunch break or tonight. Am on UTC+1)

**Comment by tfsomrat** [comment]
I am using ZSH, but I tried on default terminal of the mac as well

**Comment by tfsomrat** [comment]
why don't you try running `hugo mod clean --all && hugo mod get -u ./... && hugo mod tidy`

**Comment by jmMeessen** [comment]
This is indeed how I am investigating the issue (that is now confirmed to be a local issue). And the failing download is caused by the `hugo mod get...` command.
Google articles with the error message point to a GO misconfiguration. I checked this and am puzzled as I am able to program in GO (have several programs published). I am investigating a Hugo misconfiguration: up to now I used a containerized Hugo distribution.

I am now a big step forward and know that the problem is local to my two laptops. But still scratching my head....
